Update ONIX Source page for v0.2.0

bparks13 opened the pull request:
This PR adds and updates information for the ONIX Source plugin so that it is consistent with v0.2.0. 

Notable updates:

- Automated search for calibration files for all Neuropixels devices
- Separated AuxiliaryIO into DigitalIO and AnalogIO tabs, as well as separating the data streams

Version 0.2.0 also includes some bug fixes, but those are not touched upon in the documentation. For more details on this release, check out the release page on the onix-source repo.

cjsha commented on source/User-Manual/Plugins/Onix-Source.rst:
```diff
@@ -287,32 +307,40 @@ breakout board. The breakout board is the main hub for the ONIX system, and it i
-events are sampled at 25 kHz, and can be used to record button presses or digital inputs on the
-breakout board. The first 8 digital channels record the digital inputs, and the last 6 digital
-channels record the button presses.
+Digital data is saved as both continuous channels and events, and can be visualized using the "LFP
```
I would usually put a reference to the LFP Viewer plugin page here

cjsha commented:
If [onix-source/#126](https://github.com/open-ephys-plugins/onix-source/issues/126) will be implemented 0.2.0, does it need to be documented here?

cjsha commented:
I think it's worth giving an admonition about how memory usage shouldn't climb as shown in the screenshot and should stay at/return to zero or else the acquisition session will terminate in an error.

bparks13 commented:
@cjsha Thanks for the suggestions! I've gone through and added them here, let me know fi you think there are any other comments or if this can be merged.

Thank you for pointing out that the milestone was not fully closed, I have moved the remaining issues to the next release so that we can get v0.2.0 out sooner.
